### PR TITLE
fix: Avoid token typos

### PR DESCRIPTION
**Pull request by slavek-kucera**

### CODE REVIEW

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDefineOptionsCheckUtility.java [hunk 1]
```diff
@@ -32,50 +33,50 @@ public class CICSDefineOptionsCheckUtility extends CICSOptionsCheckBaseUtility {
...
-          put("HOURS", ErrorSeverity.ERROR);
-          put("MINUTES", ErrorSeverity.ERROR);
-          put("SECONDS", ErrorSeverity.ERROR);
-          put("YEAR", ErrorSeverity.ERROR);
-          put("MONTH", ErrorSeverity.ERROR);
-          put("DAYOFMONTH", ErrorSeverity.ERROR);
-          put("DAYOFYEAR", ErrorSeverity.ERROR);
-          put("AFTER", ErrorSeverity.ERROR);
-          put("AT", ErrorSeverity.ERROR);
-          put("0N", ErrorSeverity.ERROR);
```
Comment (slavek-kucera) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDefineOptionsCheckUtility.java:78]: here

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSExtractOptionsUtility.java [hunk 2]
```diff
@@ -32,95 +33,95 @@ public class CICSExtractOptionsUtility extends CICSOptionsCheckBaseUtility {
...
-          put("IUTYPE", ErrorSeverity.ERROR);
-          put("DATASTR", ErrorSeverity.ERROR);
-          put("RECFM", ErrorSeverity.ERROR);
-          put("ATTRIBUTES", ErrorSeverity.ERROR);
-          put("STATE", ErrorSeverity.ERROR);
-          put("CERTIFICATE", ErrorSeverity.ERROR);
-          put("LENGTH", ErrorSeverity.ERROR);
-          put("SERIALNUMLEN", ErrorSeverity.ERROR);
-          put("USERID", ErrorSeverity.ERROR);
-          put("COMMONNAMELEN", ErrorSeverity.ERROR);
```
Comment (slavek-kucera) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSExtractOptionsUtility.java:57]: here

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSExtractOptionsUtility.java [hunk 3]
```diff
@@ -32,95 +33,95 @@ public class CICSExtractOptionsUtility extends CICSOptionsCheckBaseUtility {
...
-          put("OWNER", ErrorSeverity.WARNING);
-          put("ISSUER", ErrorSeverity.WARNING);
-          put("LOGONMSG", ErrorSeverity.ERROR);
-          put("INTO", ErrorSeverity.ERROR);
-          put("SET", ErrorSeverity.ERROR);
-          put("PROCNAME", ErrorSeverity.ERROR);
-          put("PROCLENGTH", ErrorSeverity.ERROR);
-          put("MAXPROCLEN", ErrorSeverity.ERROR);
-          put("SYNCLEVEL", ErrorSeverity.ERROR);
-          put("PIPELENGTH", ErrorSeverity.ERROR);
```
Comment (slavek-kucera) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSExtractOptionsUtility.java:78]: here

**Comment by slavek-kucera** [comment]
@chacebot

**Review by amanPras — APPROVED** [review]

**Pull request merged**